### PR TITLE
Oss CI Export

### PR DESCRIPTION
**Pull request by winkingturtle-vmw**
Use [wg-app-platform-runtime-ci](https://github.com/cloudfoundry/wg-app-platform-runtime-ci) for testing an releasing this pipeline. 

- Add the ability to version released-binaries so that when i run `cf plugins` I see a version that matches github releases.
- Unify docker workflow with the remaining releases under runtime

### CODE REVIEW

File: scripts/docker/build-binaries.bash [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+pushd /repo > /dev/null
+git_configure_safe_directory
+REPO_NAME=$(git_get_remote_name)
+popd > /dev/null
+
+. <(/ci/shared/helpers/extract-default-params-for-task.bash /ci/shared/tasks/build-binaries/linux.yml)
+
+export DEFAULT_PARAMS="/ci/$REPO_NAME/default-params/build-binaries/linux.yml"
+pushd / > /dev/null
+/ci/shared/tasks/build-binaries/task.bash
```
Comment (reneighbor) [inline scripts/docker/build-binaries.bash:17]: @winkingturtle-vmw Hi! I'm running this task in a docker container, but the build-binaries task is getting an unbound variable error on `$MAPPING`. 
```
root@55a995012ac3:/# repo/scripts/docker/build-binaries.bash
/ci/shared/tasks/build-binaries/task.bash: line 22: MAPPING: unbound variable
```
Maybe it should get set here?

Comment (reneighbor) [reply scripts/docker/build-binaries.bash:17]: Nevermind, I pulled the `wg-app-platform-runtime-ci` repo and it passes!

File: plugins/app_plugin.go [hunk 2]
```diff
@@ -110,12 +117,23 @@ func (p CPUEntitlementPlugin) Run(cli plugin.CliConnection, args []string) {
...
+			minor, _ = strconv.Atoi(version[1])
+			build, _ = strconv.Atoi(version[2])
+		}
+	}
 	return plugin.PluginMetadata{
 		Name: "CPUEntitlementPlugin",
 		Version: plugin.VersionType{
-			Major: 0,
-			Minor: 0,
-			Build: 2,
```
Comment (reneighbor) [inline plugins/app_plugin.go:118]: Wow, was the plugin version hard-coded to 0.0.2 before?

**Review by reneighbor — APPROVED** [review]
Thank you for all your hard work creating / implementing standards for our CI and for providing a way to run unit AND integration tests in a controlled Docker environment! Thanks for catching and fixing the bug with `version`as well.

File: plugins/app_plugin.go (hunk 2 above)
Comment (winkingturtle-vmw) [reply plugins/app_plugin.go:118]: Yes, unfortunately.

**Pull request merged**